nodeup/pkg/model/docker.go

```diff
@@ -525,50 +525,50 @@ var dockerVersions = []dockerVersion{
 
 	// 18.03.1 - Bionic
 	{
-		DockerVersion: "18.03.1",
```

Comment:
Can we leave all the existing versions and add these new versions instead?

Reply:
Thanks @mikesplain for the prompt feedback, added 18.06.2 instead
of replacing existing entries, PTAL.